services/workflows-service/src/business-report/business-report.controller.internal.ts

```diff
@@ -350,7 +353,7 @@ export class BusinessReportControllerInternal {
     });
   }
 
-  @common.Post('/upload-batch')
+  @common.Post('/upload-batch/:type')
   @Public()
```

Comment:
Pubilc?

Reply:
I used it to test with swagger. Will remove it